src/core/CMakeLists.txt

```diff
@@ -49,6 +49,9 @@ target_include_directories(openvino_core_dev INTERFACE
     $<BUILD_INTERFACE:${OpenVINO_SOURCE_DIR}/src/common/transformations/include>
     $<BUILD_INTERFACE:${OpenVINO_SOURCE_DIR}/src/common/low_precision_transformations/include>)
 
+target_include_directories(openvino_core_dev INTERFACE
```

Comment:
you can try to add `SYSTEM` to prevent warnings from xbyak headers.

See https://cmake.org/cmake/help/latest/command/target_include_directories.html

Reply:
It works, thanks